src/client/components/app-header/app-header.vue

```diff
@@ -34,29 +35,88 @@ export default {
     locale() { return this.$i18n.locale },
     menu() { return this.menuI18n[this.locale] },
   },
+
+  methods: {
```

Comment:
I would keep "handling scroll" out of this PR entirely. In a Vue or React or other SPA framework, never use `.getElementById` or any DOM selector and never user `.classList. ...`. These are things you handle in the template. So instead of `appHeader.classList.add('hide')` you change a state of the component, like `this.isHidden = true`. Then inside of the template on the element with `#app-header` you use `v-if` or a conditional `:class` and maybe some Vue animation. Also scroll listeners should be passive. And in addition I think we can use native sticky or similar behaviours. So let's discuss this a bit more in person, but leave it out of this PR.